modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/DateFormatTests.java

```diff
@@ -93,6 +93,15 @@ public void testParseISO8601() {
                 equalTo("2001-01-01T08:00Z"));
     }
 
+    public void testParseWhenZoneNotPresentInText() {
+        assertThat(DateFormat.Iso8601.getFunction(null, ZoneOffset.of("+0100"), null).apply("2001-01-01T00:00:00").toInstant().toEpochMilli(),
+            equalTo(978303600000L));
+        assertThat(DateFormat.Iso8601.getFunction(null, ZoneOffset.of("+0100"), null).apply("2001-01-01T00:00:00").toString(),
+            equalTo("2001-01-01T00:00+01:00"));
+        assertThat(DateFormat.Iso8601.getFunction(null, ZoneOffset.of("+0100"), null).apply("2001-01-01T00:00:00").toString(),
```

Comment:
This line appears to be identical to the preceding line? Am I missing something?